src/main/osd/osd_elements.c

```diff
@@ -1034,6 +1034,18 @@ static void osdElementLinkQuality(osdElementParms_t *element)
         tfp_sprintf(element->buff, "%c%1d", SYM_LINK_QUALITY, osdLinkQuality);
     }
 }
+
+static void osdElementTxUplinkPower(osdElementParms_t *element)
+{
+    if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) {
+        const uint16_t osdUplinkTxPower = rxGetUplinkTxPwr();
+        if (osdUplinkTxPower < 1000) {
+            tfp_sprintf(element->buff, "%c%3dMW", SYM_RSSI, osdUplinkTxPower);
+        } else {
+            tfp_sprintf(element->buff, "%c%dW", SYM_RSSI, osdUplinkTxPower / 1000);
```

Comment:
It's a bad assumption that any value above 1000mw will be an integer multiple of 1000. What if there's a new module or a firmware update that enables 1500mw? Instead suggest:
```
    osdPrintFloat(element->buff, SYM_RSSI, osdUplinkTxPower / 1000.0f, "", 1, false, 'W');
```